src/docs/sdk/performance/modules/assets.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Assets Module'
+---
+
+The javascript SDK should automatically instrument most assets fetched by the application. This includes images, stylesheets, scripts, and other assets. But if you may wish to instrument assets manually.
+
+## Span conventions
+
+### Span Operations
+
+| Span OP           | Description                             |
+| :---------------- | :-------------------------------------- |
+| `resource.script` | Javscript was fetched by the browser    |
+| `resource.css`    | A Stylesheet was fetched by the browser |
+| `resource.img`    | An Image was fetched by the browser     |
+
+### Span Description
+
+The description of an asset span should be the full URL which is was fetched from, e.g. `https://cdn.com/main.hash123.js`.
+
+### Span Data
```

Comment:
This section should link to the "Span Data Conventions" section for resources. That way you don't have to re-do the table. You just need to explain which of those attribute are mandatory (all of them?)

Reply:
🤔 There is no "resource" section in span data conventions. Resource spans contains some properties from the [browser section](https://develop.sentry.dev/sdk/performance/span-data-conventions/#browser) and some properties from the [ http section ](https://develop.sentry.dev/sdk/performance/span-data-conventions/#http)